packages/plugin-auto-tracking-browser/src/libs/finder.ts

```diff
@@ -0,0 +1,322 @@
+/* istanbul ignore file */
```

Comment:
Can we provide some explanation for this helper function?

Reply:
Yes, there is a detailed doc here: https://github.com/antonmedv/finder. In brief, we use this helper function to get a unique CSS selector of the element (in current page).